spec/spec_helper_acceptance.rb

```diff
@@ -45,7 +45,7 @@ def install_modules_on(host)
     master['puppetservice'] = 'puppetserver'
     master['puppetserver-confdir'] = '/etc/puppetlabs/puppetserver/conf.d'
     master['type'] = 'aio'
-    install_puppet_agent_on master, {:puppet_agent_version => '1.2.1'}
+    install_puppet_agent_on master, {}
```

Comment:
Yes, because the description did not match the actual call signature. See https://github.com/puppetlabs/beaker/pull/1080.